docs/admin-guide/README.md

```diff
@@ -0,0 +1,15 @@
+# Administrator guide - Keptn Lifecycle Controller
+
+Documentation for administrators who install and maintain the Keptn Lifecycle Controller:
+Installation guide, Upgrading Keptn, Architecture, Troubleshooting, etc.
+
+>
+> **_NOTE:_**  This section is under development
+> 
+
+## Contents
+
+- [Installing Developer Builds](./install-dev-build.md)
```

Comment:
This is more part of the contributor guide than the admin guide. I would not encourage an admin to install a dev-build.